etc/kayobe/inventory/group_vars/all/dev-tools

```diff
@@ -0,0 +1,8 @@
+---
+########################################################################
+# List of development tool packages to install during host configuration
+dev_tools_packages_extra:
+  - tcpdump
+  - jq
+  - wget
```

Comment:
Is it really useful when `curl` is already installed?

Reply:
I've seen a fair number of errors from scripts and things that just assume it will exist.